### PR TITLE
E-gun/taser spritey stuff, .45 c-20r, etc.

### PR DESCRIPTION
**Pull request by ghost**
- Better frankensteined egun sprites
- Temp gun, xray laser, and botany gun have proper 'empty' animations
- Proper inhands for the egun and taser that reflect their new sprites (+empty anims)
- C20r is now a .45 smg instead of ridiculously stupid 12mm (it has the same stats anyway)
  - Fixes cigarettes having missing sprites on the inhands (blinking invisible randomly when they're in your hand)

Remember kids, there's no excuse for *merging lazy, unfinished PRs!

### CODE REVIEW

**Comment by JStheguy** [comment]
Why make the lights on the guns blink red and green when empty?
Why not just solid/blinking red?

**Comment by ghost** [comment]
Because that's how the ion rifle has done it for literally ages

Besides, it looks 100$ better than straight blinking. If you want me to get into some serious aesthetic rant about it, it's much more eye catching to flip from opposite colors, especially on a non-perfect loop or what have you. You can notice you're out of ammo out of your peripheral, or by your inhand, MUCH faster. I'd prefer it if you'd notice how quite literally every other laser has an empty anim. I meant to implement it for the egun as well but you beat me to it with these resprite crap so I had to redo all of my (and your) work.

**Comment by ghost** [comment]
Lel I forgot to change the title

**Comment by JStheguy** [comment]
ss13shuttle

**Comment by ghost** [comment]
Yes it's also the first commit's name well done JS mission accomplished you proved you can take screenshots as well as be lazy with sprite changes

**Comment by ghost** [comment]
Added silenced c20r sprites that will be used once that silencer proc is fixed!

Gonna tag this as WIP until the silencer proc is finished because I might as well include that in this too.

**Comment by ghost** [comment]
Done. C-20r is silencer-compatible, as are any future weapons and some descriptions have been changed around a bit.

**Comment by ghost** [comment]
BEFORE SOMEONE ACCUSES ME OF STEALTH NERF THE SYNDIE BORGS ARE 26 TC NOW SO YOU CAN'T BUY TWO WITH 20 READIES, YOU NEED POP SCALE.

File: config/admins.txt [hunk 1]
```diff
@@ -60,3 +60,4 @@ yota = Game Master
 firecage = Game Master
 donkieyo = Game Master
 argoneus = Game Master
+paprka = Game Master
```
Comment (ghost) [inline config/admins.txt:63]: fuck w/ it

**Comment by Atskadan** [comment]
muh laziness!!!

**Pull request merged**